samples/AlarmBot/AlarmBot.cs

```diff
@@ -13,11 +13,10 @@ public class AlarmBot : IBot
     {
         public async Task OnReceiveActivity(IBotContext botContext)
         {
+            // Get the current ActiveTopic from my persisted conversation state
             var context = new AlarmBotContext(botContext);
 
-            // --- Our receive handler simply inspects the persisted ITopic class and calls to it as appropriate ---
-
-            bool handled = false;
+            var handled = false;

```

Comment:
What am I missing here? This variable seems like it's not used at all...

Reply:
I asked the same question myself! It is in fact used, line 35, to switch back to default topic:
`if (handled == false && !(context.ConversationState.ActiveTopic is DefaultTopic))`